docs/usage/usage.md

```diff
@@ -234,6 +234,52 @@ all_results = dl.multilevel_modeling(ct_subs=ct_subs,
                                      )
 ```
 
+### Distances and Permutation Tests
+General purpose functions for distances and permutation tests. Reimplements
+functions from [scperturb](http://projects.sanderlab.org/scperturb/) package.
+
+```{eval-rst}
+.. currentmodule:: pertpy
+```
+
+```{eval-rst}
+.. autosummary::
+    :toctree: tools
+
+    tools.Distances
+    tools.PermutationTest
+```
+
+See [Distance tutorial](https://pertpy.readthedocs.io/en/latest/tutorials/notebooks/distances.html) 
+and [Permutation test tutorial](https://pertpy.readthedocs.io/en/latest/tutorials/notebooks/distance_tests.html) for a more elaborate tutorial.
+
+```python
+import pertpy as pt
+import scanpy as sc
+
+adata = pt.dt.dixit_2016_scperturb()
+
+# basic qc and pp
```

Comment:
Do you think that we can use a dataset that requires less preprocessing? I'd like to keep these examples here as minimal as possible. If required we can also upload a "preprocessed" version of the dataset.

Reply:
Definitely possible. Where should I put that dataset and the code to produce it?

Reply:
https://github.com/theislab/pertpy-datasets

Then I upload it to figshare and write the dataloader for it